### PR TITLE
Ci/automate esm tests info update

### PR DESCRIPTION
**Pull request by mandresm**
Should allow for approval of changes in the esm_tests info by typing $approve-changes command (change $ by #).

### CODE REVIEW

**Comment by mandresm** [comment]
a comment

**Comment by mandresm** [comment]
#approve-changes

**Comment by mandresm** [comment]
#approve-changes

**Comment by pgierz** [comment]
The second time did not trigger a new commit because it is already up to date...? Is that correct?

**Comment by mandresm** [comment]
I thought the second time triggered as well, let's see if it does it now

**Comment by mandresm** [comment]
#approve-changes

**Comment by mandresm** [comment]
yeap, it's working. Is not pushing because I leaft that change for the end, I don't want to add the pushing step until I'm sure I'm not destroying the last_tested info

**Comment by pgierz** [comment]
`last_tested` is a submodule though, right? so if you accidentally nuke it, one can go back in time, right?

**Comment by mandresm** [comment]
yeah, of course, nothing to be so worry about, I just try to save on things I destroy so that I have to type less XD

**Comment by mandresm** [comment]
#approve-changes...

**Comment by mandresm** [comment]
Okay, so the CI seems to always get the release version of the workflows, and I'm kind of tired of digging into the documentation of GitHub Actions. I have completed the actions to now push and commit, so the feature should be complete and ready to merge. I cannot test it to it's full... but as @pgierz, it can only break the `last_tested` repo and that shouldn't be a problem at all...

I will merge without bumping, if that's okay with everyone since this is CI stuff

PD: I've done my best to keep the yaml files for the workflows compact and not repetitive, but GitHub Actions syntax has its limitations and my patience too...

File: .github/workflows/approve_esm_tests_changes.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ on:
     
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
-
+
```
Comment (mandresm) [inline .github/workflows/approve_esm_tests_changes.yml:16]:
```suggestion
 
```

**Review by pgierz — APPROVED** [review]

**Pull request merged**